pom.xml

```diff
@@ -10,7 +10,7 @@
 
   <groupId>org.sonarsource.scanner.cli</groupId>
   <artifactId>sonar-scanner-cli</artifactId>
-  <version>6.1-SNAPSHOT</version>
+  <version>6.0.1-SNAPSHOT</version>
```

Comment:
Just checking: what is the rationale for going to 6.0.1 instead of 6.1?

Reply:
I was about to release a patch. But in the end, this is not needed. I will drop this commit.